src/github/utils/plugins.ts

```diff
@@ -17,8 +17,8 @@ export function getManifest(context: GitHubContext, plugin: string | GithubPlugi
   return isGithubPlugin(plugin) ? fetchActionManifest(context, plugin) : fetchWorkerManifest(plugin);
 }
 
-async function fetchActionManifest(context: GitHubContext<"issue_comment.created">, { owner, repo }: GithubPlugin): Promise<Manifest | null> {
-  const manifestKey = `${owner}:${repo}`;
+async function fetchActionManifest(context: GitHubContext<"issue_comment.created">, { owner, repo, ref }: GithubPlugin): Promise<Manifest | null> {
+  const manifestKey = `${owner}:${repo}:${ref ?? ""}`;
```

Comment:
Empty string always seems like a bad idea. This seems to be extremely error prone compared to explicitly handling this with `null` and skipping the subsequent code blocks that use this value.